ci-operator/step-registry/cucushift/installer/check/nutanix/multi-disks/cucushift-installer-check-nutanix-multi-disks-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+check_result=0
+
+if [[ -f ${CLUSTER_PROFILE_DIR}/secrets.sh ]]; then
+    NUTANIX_AUTH_PATH=${CLUSTER_PROFILE_DIR}/secrets.sh
+else
+    NUTANIX_AUTH_PATH=/var/run/vault/nutanix/secrets.sh
+fi
+declare prism_central_host
+declare prism_central_port
+declare prism_central_username
+declare prism_central_password
+# shellcheck disable=SC1090
+source "${NUTANIX_AUTH_PATH}"
+
+pc_url="https://${prism_central_host}:${prism_central_port}"
+api_ep="${pc_url}/api/nutanix/v3/vms/list"
+un="${prism_central_username}"
+pw="${prism_central_password}"
+
+function check_multi_disks() {
+    data="{
+        \"filter\":\"vm_name==$node\"
+    }"
+    node_json=$(curl -ks -u "${un}":"${pw}" -X POST "${api_ep}" -H "Content-Type: application/json" -d @- <<<"${data}")
+    disk_adapter_type=$(echo "${node_json}" | jq -r ".entities[].status.resources.disk_list[] | select(.disk_size_mib==$(echo "$DATA_DISK_SIZE" | tr -d "Mi")) | .device_properties.disk_address.adapter_type")
+
+    if [[ $disk_adapter_type == "$DATA_DISK_ADAPTER_TYPE" ]]; then
+        echo "Pass: passed to check node multi_disks"
+    else
+        echo "Fail: failed to check node multi_disks"
+        check_result=$((check_result + 1))
+    fi
+}
+
+IFS=' ' read -r -a worker_nodes_list <<<"$(oc get nodes -l node-role.kubernetes.io/worker= -ojson | jq -r '.items[].metadata.name' | xargs)"
+
+for node in "${worker_nodes_list[@]}"; do
+    check_multi_disks "$node"
```

Comment:
btw, once the extra disk is mounted to vms, what it will be used for ?

Reply:
That depends on customer I think

Reply:
So currently it is not used yet, even not mounted to any system file, right?

Reply:
yes